src/main/java/com/pet/bankservice/entity/Role.java

```diff
@@ -0,0 +1,26 @@
+package com.pet.bankservice.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "roles")
+public class Role {
+    public enum RoleName {
+        ADMIN,
+        USER;
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(EnumType.STRING)
+    private RoleName roleName;
```

Comment:
Make it unique, you don't need 15 same ADMIN's

Reply:
done